### DISCUSSION TITLE
Do I need special machine? No and Yes

### DISCUSSION THREAD

**Discussion by leeyemi** [tearust/teaproject#7] [Q&A] [answered]
Hi there, 
I read this statement from the teaproject site (https://teaproject.org/#/doc_list/%2FWhitepaper%2FAppendix_A.md)
"As your stake scales, and you are able to make an initial return on investment, you have to be proactive in investing in stronger GPUs** and more robust mining infrastructure so that you can continue to scale along with your CML".

Appreciate for further explanation related to GPU enhancement required over time.

Thanks you

**Comment by kevingzhang** [comment, marked as answer]
TEA mining need a machine with TPM chip. You can think this is a special requirement, but if you know every computer produced after 2010 will have a type of TPM chip, you won't consider this is a special machine.

Window 11 will not only require a TPM chip, but also require a TPM 2.0 chip. This is exactly the same requirement as TEA mining machine. 

So do you think to consider the TEA miner is a special machine?

Regarding the strong GPU/CPU/TPU, that are requirements to run special requirements from the TApps (Application running in T is called TApps, they are Rich dDapps). Most AI related applications require GPU or TPU. That is not for TEA project use.
